modules/appeals_api/app/models/appeals_api/non_veteran_claimant.rb

```diff
@@ -0,0 +1,155 @@
+# frozen_string_literal: true
+
+module AppealsApi
+  class NonVeteranClaimant
+    def initialize(auth_headers:, form_data:)
+      @auth_headers = auth_headers
+      @form_data = form_data
+    end
+
+    def first_name
+      auth_headers['X-VA-Claimant-First-Name']
+    end
+
+    def middle_initial
+      auth_headers['X-VA-Claimant-Middle-Initial'] || ''
```

Comment:
Why all the blank string returns? I thought pdftk did the translation from nil values into a blank string for us?

Reply:
I thought it didn't! I'm looking into this!